tests/integration/lpt/test_lpt.py

```diff
@@ -233,3 +231,27 @@ def test_aggregate_holdings(self):
             lambda left: self.fail(lpt.display_error(left)),
             lambda right: self.assertIsInstance(right, DataFrame),
         )
+
+    def test_aggregate_group_holdings(self):
+
+        if not self.target_portfolios_exist():
+            self.skipTest("missing target portfolios")
+
+        qah.process_args(
+            self.api,
+            qah.parse(
+                args=[
+                    "Finbourne-Examples",
+                    "US-FI",
```

Comment:
Should there be a statement like self.target_portfolios_exist() to assert that this group exists?

Reply:
good spot, added